### ISSUE TITLE
De-duplicate list attribute values

### ISSUE THREAD

**Issue by stevenvachon** [posthtml/htmlnano#39]
such as `class`, `ping` and `rel`.

```html
<a class="class1   class2 class2" ping="http://domain1.com, http://domain2.com" rel="me me   external">
```
would compile to:
```html
<a class="class1 class2" ping="http://domain1.com,http://domain2.com" rel="me external">
```

These would be handy: [list-to-array](https://npmjs.com/list-to-array), [dupe](https://npmjs.com/dupe)

If we wanted to de-dupe URL lists (such as `ping`), we could use [specurl](https://github.com/stevenvachon/specurl)'s `relation()`.

**Comment by maltsev** [comment]
For now I only implemented `class`, `rel`, and `ping` minification as `deduplicateAttributeValues` module. I'm going to release it in version 0.2.0.

**Comment by maltsev** [comment]
Released: [0.2.0](https://github.com/posthtml/htmlnano/releases/tag/0.2.0).
